From c8e5e6e7323642f7e6f12ee5f5231b0ec44c40ab Mon Sep 17 00:00:00 2001
From: Dave Reisner <dreisner@archlinux.org>
Date: Thu, 21 Nov 2013 12:25:27 -0500
Subject: [PATCH] pylibmount: correctly import from pylibmount.so

Without this, python is unable to find the module:

$ python -c 'import libmount'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/lib/python3.3/site-packages/libmount/__init__.py", line 1, in <module>
    from pylibmount import *
ImportError: No module named 'pylibmount'

Signed-off-by: Dave Reisner <dreisner@archlinux.org>
---
 libmount/python/libmount/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libmount/python/libmount/__init__.py b/libmount/python/libmount/__init__.py
index 243c639..09104e2 100644
--- a/libmount/python/libmount/__init__.py
+++ b/libmount/python/libmount/__init__.py
@@ -1,2 +1,2 @@
-from pylibmount import *
+from .pylibmount import *
 
-- 
1.8.4.2

